config/manifests/ci.yaml

```diff
@@ -7,3 +7,4 @@ reviewers:
   - f41gh7
   - AndrewChubatiuk
   - Haleygo
+  - victoriametrics-bot
```

Comment:
As far as I remember it also must be manually edited at operator-hub with a separate PR. 

This change shouldn't be included at commit with CRD changes.

Reply:
I've added user manually, in operatorhub repo, but actually it's managed with our release pipeline as well